app/components/primer/alpha/dialog.pcss

```diff
@@ -1,11 +1,5 @@
 /* Overlay */
 
-/* TODO: One day this can be :has(:modal), when it is better supported */
-body.has-modal {
-  overflow: hidden !important;
-  padding-right: var(--dialog-scrollgutter) !important;
-}
-
```

Comment:
Didn't we just add this?

Reply:
This looks like a rebase error. My bad I'll fix.